src/test/java/bio/terra/service/filedata/azure/AzureSynapsePdaoSnapshotConnectedTest.java

```diff
@@ -294,6 +294,33 @@ public void testSnapshotByQuery() throws SQLException, IOException {
     createSnapshotRowIdParquet(snapshotByQueryTableRowCounts);
   }
 
+  @Test
+  public void testQuerySnapshotWithUnderscoreInFlightId() throws SQLException, IOException {
+    String ingestFlightId = "_" + UUID.randomUUID().toString();
```

Comment:
`toString()` is unnecessary in a string concatenation expression, since all expression elements are coerced to `String`.
```suggestion
    String ingestFlightId = "_" + UUID.randomUUID();
```